frontends/web/src/locales/en/app.json

```diff
@@ -1224,7 +1224,7 @@
         "title": "Connect your own full node"
       },
       "setProxyAddress": "Set proxy address",
-      "splitAccounts": "Separate accounts by address type",
+      "splitAccounts": "Separate accounts by address type (legacy behavior)",
```

Comment:
please remember that i18n changes need to happen on locize, otherwise the changes will be lost. See https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/docs/i18n.md.

So you can remove this from the commit.